src/GraphQL/Introspection/__Type.cs

```diff
@@ -38,7 +38,7 @@ public __Type(bool allowAppliedDirectives = false)
 
             Field<StringGraphType>("name").Resolve(context => context.Source!.Name);
 
-            Field<StringGraphType>("description");
+            Field<StringGraphType>("description").Resolve(context => context.Source!.Description);
```

Comment:
```suggestion
            Field<StringGraphType>("description").Resolve(context => context.Source!.Description); // avoid implicit use of FieldNameResolver here to make the code compatible with AOT compilation
```

Reply:
I don't think we should litter the code with these comments.  This is true for almost every use of Resolve within the introspection types.  For example, why here and not two lines above which is basically identical?

Note that originally NameFieldResolver was used everywhere, but we changed it to use Resolve was so that execution was faster.  We must have missed a couple instances.

Reply:
> This is true for almost every use of Resolve within the introspection types.

Yes. Do as you like. For me personally such comments play as reminder. I tend to forgot more and more things as more and more projects I have to support.